runtime/compiler/optimizer/SwitchAnalyzer.cpp

```diff
@@ -1269,10 +1269,6 @@ TR::Block *TR::SwitchAnalyzer::addTableBlock(SwitchInfo *dense)
       node->setAndIncChild(0, TR::Node::create(TR::isub, 2,
                                            (_isInt64 ? TR::Node::create(TR::l2i, 1, TR::Node::createLoad(_switch, _temp)) : TR::Node::createLoad(_switch, _temp)),
                                            TR::Node::create(_switch, TR::iconst, 0, dense->_min)));
-   else
-      node->setAndIncChild(0, TR::Node::create(TR::iusub, 2,
-                                           (_isInt64 ? TR::Node::create(TR::l2i, 1, TR::Node::createLoad(_switch, _temp)) : TR::Node::createLoad(_switch, _temp)),
-                                           TR::Node::create(_switch, TR::iuconst, 0, dense->_min)));
```

Comment:
The corresponding code just above shouldn't be guarded by `_signed` anymore. The only difference between the `_signed` and `!_signed` cases was the choice of `isub` vs. `iusub` and of `iconst` vs. `iuconst`. Substituting in the sign-agnostic opcodes for the unsigned ones, both branches are the same, so the one currently under `if (_signed)` should run whether or not `_signed` is set